if there's no fonts.conf in either ~/.gimp-2.8 or /etc/gimp/2.0 gimp loads
no font at all. fall back to use fontconfig's system fonts instead.

--- a/app/text/gimp-fonts.c
+++ b/app/text/gimp-fonts.c
@@ -114,16 +114,17 @@
 
   fonts_conf = gimp_personal_rc_file (CONF_FNAME);
   if (! gimp_fonts_load_fonts_conf (config, fonts_conf))
-    goto cleanup;
+    goto load;
 
   fonts_conf = g_build_filename (gimp_sysconf_directory (), CONF_FNAME, NULL);
   if (! gimp_fonts_load_fonts_conf (config, fonts_conf))
-    goto cleanup;
+    goto load;
 
   path = gimp_config_path_expand (gimp->config->font_path, TRUE, NULL);
   gimp_fonts_add_directories (config, path);
   g_free (path);
 
+load:;
   if (status_callback)
     {
       gint64                 end_time;
@@ -167,6 +168,7 @@
   gimp_font_list_restore (GIMP_FONT_LIST (gimp->fonts));
 
  cleanup:
+  if(config) FcConfigDestroy (config);
   gimp_container_thaw (GIMP_CONTAINER (gimp->fonts));
   gimp_unset_busy (gimp);
 }
@@ -197,7 +199,6 @@
 
   if (! FcConfigParseAndLoad (config, (const guchar *) fonts_conf, FcFalse))
     {
-      FcConfigDestroy (config);
       ret = FALSE;
     }
 
